qiita_pet/templates/edit_study.html

```diff
@@ -39,6 +40,17 @@
       // removing white spaces from the front of the text
       $(this).val(title.trimLeft());
     });
+
+    $("#new_person_name").on("keyup", function() {
+      var charlie  = $(this).val();
```

Comment:
mhhh ... not sure if I agree with the name of the variable ...

Reply:
doh! My apologies, it was guaranteed to be a unique name; I forgot to make it a more appropriate one. :)